src/entities/Route/types.ts

```diff
@@ -71,8 +71,8 @@ export const defaultOrigin = [
   /https:\/\/([a-zA-Z0-9\-_]+\.)*decentraland\.net/,
   /https:\/\/([a-zA-Z0-9\-_]+\.)*decentraland\.co/,
   /https:\/\/([a-zA-Z0-9\-_]+\.)*decentraland\.io/,
```

Comment:
Please remove `decentraland.io`, `decentraland.net` and `decentraland.co` as well